mods/boats/init.lua

```diff
@@ -229,8 +229,8 @@ minetest.register_craftitem("boats:boat", {
 		local under = pointed_thing.under
 		local node = minetest.get_node(under)
 		local udef = minetest.registered_nodes[node.name]
-		if udef and udef.on_rightclick and
-				not (placer and placer:get_player_control().sneak) then
+		if placer and udef and udef.on_rightclick and
```

Comment:
Nil placers should be allowed to place, so pass it through to on_rightclick even if placer is nil.